desktop/apps/personalize/routes.js

```diff
@@ -0,0 +1,39 @@
+import * as _ from 'underscore'
+import { renderLayout } from '@artsy/stitch'
+import App from 'desktop/apps/personalize/components/App'
+
+export const index = (req, res) => {
+  req.user.fetch({
+    success: (model, response, options) => {
+      res.locals.sd.CURRENT_USER = _.extend({}, response, res.locals.sd.CURRENT_USER)
+      res.render('template')
+    }
+  })
+}
+
+export async function newOnboarding (req, res, next) {
+  if (res.locals.sd.ONBOARDING_TEST === 'experiment') {
+    try {
+      const layout = await renderLayout({
+        basePath: req.app.get('views'),
+        config: {
+          styledComponents: true
+        },
+        layout: '../../components/main_layout/templates/react_blank_index.jade',
+        blocks: {
+          body: App
+        },
+        locals: {
+          ...res.locals,
+          assetPackage: 'personalize'
+        }
+      })
+
+      res.send(layout)
+    } catch (error) {
+      next(error)
+    }
+  } else {
+    res.redirect('/personalize')
```

Comment:
The plan is that when we are ready to actually enable the a/b test we'd replace this else block with what's currently in the `index` method and rename this `newOnboarding` method to be `index`.

Reply:
👍 makes sense